docs/docs/quickstart.md

```diff
@@ -2,21 +2,24 @@
 
 ## Dependencies
 
-http4s is available for Scala 2.12, 2.13, and 3.
-
-You can add http4s to your build by adding its modules to your dependencies in `build.sbt`.
-Here are some common dependencies, more are available on the [integrations page][integrations].
+http4s is available for Scala 2.12, 2.13, and 3. You can add http4s to your build by adding its modules to `libraryDependencies` in your `build.sbt`.
```

Comment:
Just wondering, is Scala 3 users so familiar with brand-new behavior of absence of forward compatibility? Shouldn't we try to cut down the cognitive load for newcomers by taking the process debt of updating docs when a new minor version of Scala 3 is dropped?

Reply:
Yeah, it's a good point, I have seen confusion here. So I will change it to say `3.2+`. Maybe when 3.3 drops, and it is an LTS, we will stay on it for a while ... but remains to be seen.

Also, Scala 2.13 might be dropping forward-compatibility as well soon, to add to the confusion 😅 
- https://github.com/scala/improvement-proposals/pull/54